data/strings/strings.txt

```diff
@@ -26894,3 +26894,390 @@
+    ro = Știți că puteți atinge lung orice loc de pe hartă pentru a-l selecta?
+    ru = Знаете ли вы, что любое место на карте можно выбрать, подержав там палец в течение секунды?
+    sk = Viete, že na ľubovoľné miesto na mape ho môžete vybrať dlhým klepnutím?
+    sv = Vet du att du kan trycka länge på vilken plats som helst på kartan för att välja den?
+    sw = Je, unajua kwamba unaweza kugonga kwa muda mrefu sehemu yoyote kwenye ramani ili kuichagua?
+    th = คุณรู้ไหมว่าคุณสามารถแตะสถานที่ใดก็ได้บนแผนที่ค้างไว้เพื่อเลือกได้
+    tr = Haritadaki herhangi bir yeri seçmek için uzun süre dokunabileceğinizi biliyor musunuz?
+    uk = Чи знаєте ви, що ви можете довго торкатися будь-якого місця на карті, щоб вибрати його?
+    vi = Bạn có biết rằng bạn có thể nhấn và giữ bất kỳ địa điểm nào trên bản đồ để chọn địa điểm đó không?
+    zh-Hans = 您知道您可以长按地图上的任何位置来选择它吗？
+    zh-Hant = 您知道您可以長按地圖上的任何位置來選擇它嗎？
+
+  [app_tip_05]
+    comment = App tip #05
+    tags = android
+    en = Did you know that your current location on the map can be selected?
+    ar = هل تعلم أنه يمكنك تحديد موقعك الحالي على الخريطة؟
+    be = А вы ведалі, што сваё месцазнаходжанне на мапе можна абраць?
+    bg = Знаете ли, че можете да изберете текущото си местоположение на картата?
+    ca = Sabíeu que podeu seleccionar la vostra ubicació actual al mapa?
+    cs = Víte, že si na mapě můžete vybrat svou aktuální polohu?
+    da = Ved du, at du kan vælge din nuværende placering på kortet?
+    de = Wussten Sie, dass Sie Ihren aktuellen Standort auf der Karte auswählen können?
+    el = Γνωρίζετε ότι μπορείτε να επιλέξετε την τρέχουσα τοποθεσία σας στο χάρτη;
+    es = ¿Sabía que su ubicación actual en el mapa se puede seleccionar?
+    et = Kas tead, et saad kaardil valida oma praeguse asukoha?
+    eu = Ba al dakizu zure uneko kokapena mapan hauta dezakezula?
+    fa = آیا می دانید که می توانید مکان فعلی خود را روی نقشه انتخاب کنید؟
+    fi = Tiesitkö, että voit valita nykyisen sijaintisi kartalta?
+    fr = Savez-vous que vous pouvez sélectionner votre position actuelle sur la carte ?
+    he = האם אתה יודע שאתה יכול לבחור את המיקום הנוכחי שלך על המפה?
+    hi = क्या आप जानते हैं कि आप मानचित्र पर अपना वर्तमान स्थान चुन सकते हैं?
+    hu = Tudja, hogy kijelölheti pozícióját a térképen?
+    id = Tahukah Anda bahwa Anda dapat memilih lokasi Anda saat ini di peta?
+    it = Sai che puoi selezionare la tua posizione attuale sulla mappa?
+    ja = 地図上で現在地を選択できることをご存知ですか?
+    ko = 지도에서 현재 위치를 선택할 수 있다는 것을 알고 계셨나요?
+    mr = तुम्हाला माहीत आहे का की तुम्ही नकाशावर तुमचे वर्तमान स्थान निवडू शकता?
+    nb = Vet du at du kan velge din nåværende posisjon på kartet?
+    nl = Wist u dat u uw huidige locatie op de kaart kunt selecteren?
+    pl = Czy wiesz, że możesz wybrać swoją aktualną lokalizację na mapie?
+    pt = Sabia que pode selecionar sua posição no mapa?
+    pt-BR = Você sabia que pode selecionar sua posição no mapa?
```

Comment:
```suggestion
    pt = Sabia que pode sua posição atual no mapa pode ser selecionada?
    pt-BR = Você sabia que pode sua posição atual no mapa pode ser selecionada?
```

Reply:
Thanks @matheusgomesms!! Can you please double-check these translations? I don't know Portuguese, but for me it looks like the verb "pode" is repeated in both sentences... Maybe they are correct, but shouldn't these translations be:
pt = Sabia que sua posição atual no mapa pode ser selecionada?
pt-BR = Você sabia que sua posição atual no mapa pode ser selecionada?

Reply:
True, thanks for the catch!

Reply:
Thanks for the confirmation.... translation changed!!